CHANGELOG.md

```diff
@@ -2,10 +2,16 @@
 
 ## Unreleased
 
+## 🚀 New components 🚀
+
+- Add memory ballast extension (#1958, #1959) 
+
 ## 🛑 Breaking changes 🛑
 
 - Use consumer for sender interface, remove unnecessary receiver address from Runner #1941
 - Enable sending queue by default in all exporters configured to use it #1924
+- Remove "mem-ballast-size-mib" flag, memory ballast can be configured as an extension.
```

Comment:
I am not sure we want this breaking change. This is user-facing breaking change. Can we avoid it? Can we give warning in the log and automatically inject the extension for now? And then we will remove in a few months.

Reply:
I can do that.